README.md

```diff
@@ -255,7 +255,7 @@ IoTDB> SELECT status FROM root.ln.wf01.wt01
 Total line number = 2
 ```
 
-We can also query several timeseries data at once like this:
+We can also query several timeseries data at the same time:
```

Comment:
how about "We can also query several timeseries data using one SQL statement"

Reply:
Yeah, I think this clearly states the purpose of the command.